### PR TITLE
Updated all types where possible in the src/ dir

### PR DESCRIPTION
**Pull request by ebebbington**
Implements #115

### CODE REVIEW

File: src/http/server.ts [hunk 1]
```diff
@@ -142,6 +156,7 @@ export default class Server {
     if (configs.static_paths) {
       this.directory = configs.directory; // blow up if this doesn't exist
       configs.static_paths.forEach(path => {
+        console.log(path)
```
Comment (player-4) [inline src/http/server.ts]: we love console logging, but this has to go away :)

Comment (ebebbington) [reply src/http/server.ts]: Noooo!! Legit can't believe I left it in :P What would be do without PRs/Code Reviews

Ill edit it now

**Comment by player-4** [comment]
@saragee3, need another reviewer

**Comment by crookse** [comment]
Hey @ebebbington,

I rebuilt the API Reference pages. I had to revert back to the old way where every class was in the sidebar. There's an issue with the Vue components not updating. Will have to look at that issue later, but your stuff is up at the following:

http://drash.land/staging

I'm going to go through and check the content in a bit. Looking good though. 👍

**Comment by crookse** [comment]
I reviewed the docs. There are some issues with the doc block parser. That's my bad (too much unstable regex parsing). I need to get the documentation repo to use the new docable code. I'll have that done tomorrow though. Once I get the docs using docable, I'll redeploy staging with the new API Reference page setup.

**Comment by crookse** [comment]
@saragee3 / @player-4, review the content please: http://drash.land/staging/

**Comment by ebebbington** [comment]
Sweet! I appreciate letting me contribute, the documentation is really impressive

**Comment by crookse** [comment]
> Sweet! I appreciate letting me contribute, the documentation is really impressive

Thank you! We're very documentation driven here :).

Also, your code is live. Thank you for this! You definitely made Drash really robust with this and improved the workflow for Drash going forward. https://github.com/drashland/deno-drash/releases

**Pull request merged**